src/replica/bulk_load/replica_bulk_loader.cpp

```diff
@@ -487,7 +488,11 @@ void replica_bulk_loader::update_bulk_load_download_progress(uint64_t file_size,
 // ThreadPool: THREAD_POOL_REPLICATION, THREAD_POOL_REPLICATION_LONG
 void replica_bulk_loader::try_decrease_bulk_load_download_count()
 {
+    if (!_is_downloading.load()) {
```

Comment:
Could this function be called concurrently in multi-thread? If it could, there is still a gap that `--_stub->_bulk_load_downloading_count;` will be executed repeatly.

Reply:
This function is run in replication thread pool, will not be called concurrently in different thread.